client/dashboard/domains/domain-forwardings/notice.tsx

```diff
@@ -0,0 +1,56 @@
+import { useSuspenseQuery } from '@tanstack/react-query';
+import { Link } from '@tanstack/react-router';
+import { Notice } from '@wordpress/components';
```

Comment:
Generally we are using a local [Notice component](https://github.com/Automattic/wp-calypso/tree/trunk/client/components/notice) in the HD. The reason was that this component was originally built for the DS, it was going to be promoted to be part of the A8C-DS until that work got paused.

Reply:
The one you linked to is the deprecated one :D But okay - I'll use the one in the dashboard/components